fedimint-ui/apps/guardian-ui/src/components/SetConfiguration.tsx

```diff
@@ -199,44 +244,59 @@ export const SetConfiguration: React.FC<Props> = ({ next }: Props) => {
               <FormHelperText>This cannot be changed later.</FormHelperText>
             </FormControl>
           </FormGroup>
-
-          <FormGroup>
-            <FormGroupHeading icon={BitcoinLogo} title='Bitcoin settings' />
-            <FormControl>
-              <FormLabel>Block confirmations</FormLabel>
-              <Input
-                type='number'
-                min={1}
-                value={blockConfirmations}
-                onChange={(ev) => {
-                  const value = ev.currentTarget.value;
-                  isValidNumber(value) && setBlockConfirmations(value);
-                }}
-              />
-              <FormHelperText>
-                How many block confirmations needed before confirming?
-              </FormHelperText>
-            </FormControl>
-            <FormControl>
-              <FormLabel>Bitcoin network</FormLabel>
-              <Select
-                placeholder='Select a network'
-                value={network !== null ? network : ''}
-                onChange={(ev) => {
-                  const value = ev.currentTarget.value;
-                  setNetwork(value as unknown as Network);
-                }}
-              >
-                {Object.entries(Network).map(([label, value]) => (
-                  <option key={value} value={value}>
-                    {label}
-                  </option>
-                ))}
-              </Select>
-            </FormControl>
-          </FormGroup>
-        </>
-      )}
+        )}
+        <FormGroup>
+          <FormGroupHeading icon={BitcoinLogo} title='Bitcoin settings' />
+          {isHost && (
+            <>
+              <FormControl>
+                <FormLabel>Block confirmations</FormLabel>
+                <Input
+                  type='number'
+                  min={1}
+                  value={blockConfirmations}
+                  onChange={(ev) => {
+                    const value = ev.currentTarget.value;
+                    isValidNumber(value) && setBlockConfirmations(value);
```

Comment:
Should `setBlockConfirmations()` run only if `isValidNumber()` is true? The `&&` here is just a boolean AND and doesn't function like in bash script.